tests/e2e/multi-apps/chainsaw-test.yaml

```diff
@@ -9,23 +9,17 @@ spec:
     - name: Prepare destination
       try:
         - script:
-            timeout: 2m
+            timeout: 5m
```

Comment:
is it really needed? can it take more than 2min??
especially since we don't wait for the destination, only `helm upgrade` it